src/routes/(auth)/api/v1/phrases/+server.ts

```diff
@@ -15,8 +15,8 @@ export const POST: RequestHandler = async (event) => {
   if (session == null) {
     throw error(403);
   }
-  const data = await request.formData();
-  const content = data.get("content")?.toString();
+  const data = await request.json();
```

Comment:
**chore:** ensure body types

Just ensure body type is right or create an issue with the todo.